server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/CreateAction.java

```diff
@@ -133,7 +133,7 @@ private CreateWsResponse doHandle(DbSession dbSession, CreateRequest createReque
         result.add(exporters.importXml(profile, importerKey, contentToImport, dbSession));
       }
     }
-    String organizationKey = qProfileWsSupport.getOrganizationKey(dbSession, result.profile());
+    String organizationKey = qProfileWsSupport.getOrganization(dbSession, result.profile()).getKey();
     dbSession.commit();
     activeRuleIndexer.index(result.getChanges());
     return buildResponse(result, organizationKey);
```

Comment:
Done - I used the organization, that has been provided as a parameter anyways...